src/phongo_bson_encode.c

```diff
@@ -130,6 +130,10 @@ static inline bool phongo_check_bson_serialize_return_type(zval* retval, zend_cl
 		return true;
 	}
 
+	phongo_throw_exception(PHONGO_ERROR_UNEXPECTED_VALUE,
+						   "Expected to receive instance of %s, %s given",
```

Comment:
Indentation looks way off here. Did you run this through `clang-format`?

Reply:
This was changed by CLion when I moved code around. `clang-format` doesn't care about these indentations, but we typically only indent an additional level for multi-line method calls. I've updated this and two other instances in this file.